src/lib_c5t_logger.h

```diff
@@ -55,8 +55,10 @@ struct C5T_LOGGER_SINGLETON_Interface {
   virtual C5T_LOGGER_Interface& operator[](std::string const& log_file_name) = 0;
 };
 
+#ifndef C5T_DLIB
```

Comment:
Enabled by now-merged https://github.com/C5T/Current/pull/1004, before this was silently OK on Linux, but failing on OS X.